### ISSUE TITLE
Gas fee field is not populated

### ISSUE THREAD

**Issue by akosenkov** [MetaMask/metamask-extension#3620]
Hi guys,

I’m trying to develop with MetaMask but not able to send a test transaction through localhost:8545.

Gas fee field is not populated (test account balance is ok though): 
Unhandled promise rejection
Error: EthQuery - RPC Error -
Stack trace:
[416]</o.prototype.sendAsync/<@moz-extension://8d28d32f-4a0a-c241-8947-f089a0e21a55/scripts/popup.js:1:658055
[1284]</o.prototype._onResponse/<@moz-extension://8d28d32f-4a0a-c241-8947-f089a0e21a55/scripts/popup.js:1:1798707
r@moz-extension://8d28d32f-4a0a-c241-8947-f089a0e21a55/scripts/popup.js:1:13664

I’m running ganache-cli on MacOSX 10.13.3 through docker: docker run -d -p 8545:8545 trufflesuite/ganache-cli:v6.1.0-beta.4
And metamask 4.2.0 through Firefox 59.0.1

Could you please help me out on what can I do?

**Comment by tmashuang** [comment]
Was trying to debug this as well. It seems when sending a transaction the `gasTotal` is not populated on the send screen. Will keep investigating this. Any insights @danjm?

**Comment by danjm** [comment]
This is a duplicate of #3391 

Users will be unblocked with https://github.com/MetaMask/metamask-extension/pull/3627

We still need to investigate the cause of the undefined RPC error from Ethquery when calling `estimateGas()`.
